include/spdk_internal/bdev.h

```diff
@@ -213,6 +213,12 @@ struct spdk_bdev_io {
 		struct {
 			enum spdk_bdev_reset_type type;
 		} reset;
+		struct {
+			void *cmd;
```

Comment:
We should probably just make a copy of the command here rather than using a pointer (so that the user doesn't have to hold on to a copy).

If we do keep this as a pointer, the lifetime requirements of cmd definitely need to be documented in the Doxygen comment for the passthru function.